ihatemoney/static/css/tagsinput.css

```diff
@@ -0,0 +1,16 @@
+.tagsinput,.tagsinput *{box-sizing:border-box}
```

Comment:
Can you put these instructions on multiple lines, rather than having each selector on a distinct line?

Reply:
I'm used to be like you and my mind has changed (I don't know if in the correct direction). I think that the functionality that this library (plugin of jquery) implements it is to easy to develop and in fact, I have read all the code in order to make sure that I can implement the behaviour that you told me. The code is straightforward and I think that the dependency should not be a problem (the code can be maintained by ourself).

This afternoon I am going to correct the css and notify you when ready.